instructionAPI/src/x86/decoder.C

```diff
@@ -0,0 +1,359 @@
+#include "x86/decoder.h"
+#include "x86/register-xlat.h"
+#include "mnemonic-xlat.h"
+#include "debug.h"
+
+#include "entryIDs.h"
+#include "registers/x86_regs.h"
+#include "registers/x86_64_regs.h"
+#include "registers/MachRegister.h"
+
+#include <cstdint>
+#include <map>
+#include <vector>
+
+namespace di = Dyninst::InstructionAPI;
```

Comment:
I'm not entirely sure.
Is this a good practice to shorten namespace like this?
Or is it better to keep them as Dyninst::instructionAPI::Result_Type?
Or maybe these are not needed cause I assume decoder is in the instructionAPI namespace?

Reply:
It's all good. I just wouldn't do this in the global namespace in a public header.